src/XeroPHP/Remote/Response.php

```diff
@@ -120,10 +120,10 @@ private function parseBadRequest(){
 			$field_errors = [];
 			foreach ($this->elements as $n => $element){
 				if (isset($element['ValidationErrors'])){
-					$errors[] = $element['ValidationErrors'][0]['Message'];
```

Comment:
This change doesn't follow the rest of the PR, but it's a minor thing I noticed whilst testing which was a bug in my previous accepted PR, so thought I may as well fix it.